UI/homepage.py

```diff
@@ -85,14 +200,33 @@ def update_items_display():
         delete_buttons.append(btn)
         y_offset += 40  # Adjust y offset for next button
 
+    # Plus Button
+    plus_button = pygame_gui.elements.UIButton(relative_rect=pygame.Rect(370, 100, 30, 30),
```

Comment:
you initalize the plus, minus, and value buttons above and set them to None, why now have this be there above, did I miss something or does it cause some sort of error?